LPS-160370 Default and System company must use the same sharded cache

achaparro opened the pull request:
Hi @tinatian,

The tests passed here:
https://github.com/liferay-uniform/liferay-portal/pull/637#issuecomment-1248610256

Can you review this pull?

Thank you.

Kvale1733 committed:
LRQA-75854 Add GetAllCompanyLogs test

Kvale1733 committed:
LRQA-75855 Add GetCompanyAdminLogs test

Kvale1733 committed:
LRQA-75858 Add GetEmptyCompanyLogs test

Kvale1733 committed:
LRQA-75859 Add AssertCompanyLogsIsolated test

Kvale1733 committed:
LRQA-75860 Add GetInvalidCompanyLogs test

Kvale1733 committed:
LRQA-75856 Add GetCompanyAdminLogsFromDifferentCompany test

Kvale1733 committed:
LRQA-75857 Add GetCompanyLogsAsGuest test

Kvale1733 committed:
LRQA-75859 Add JSON macro to get all company names

Kvale1733 committed:
LRQA-75854 Add macro to get the path for a company log

Kvale1733 committed:
LRQA-75854 Add macro to get the log file name for a company

Kvale1733 committed:
LRQA-75854 Add macro to download and return company log content

Kvale1733 committed:
LRQA-75856 Add macro to assert company log was not downloaded

Kvale1733 committed:
LRQA-75859 Add macro to assert a company log is isolated

Kvale1733 committed:
LRQA-75854 Add macro to assert company log content

vicnate5 committed:
LRQA-75854 Loop to avoid duplicate code

vicnate5 committed:
LRQA-75854 Use existing macro

vicnate5 committed:
LRQA-75854 Consistent variable naming

vicnate5 committed:
LRQA-75854 No need to create a new macro to account for default instance name

vicnate5 committed:
LRQA-75854 Add to acceptance

vicnate5 committed:
LRQA-75854 Use another method to try to assert the company ID

julschong committed:
LPS-161324 Remove SanitizerUtil from modules and use service reference instead

SelenaAungst committed:
LPS-161242 Corrected spacing between icons and link titles.

julschong committed:
LPS-161324 Remove SanitizerUtil from util-spring

julschong committed:
LPS-161324 Modify SF to make sure SanitizerUtil is no longer used in modules

p2kmgcl committed:
LPS-155518 Remove feature flag

Commit:
LPS-162241 portal-search-web: Add fields and getters and setters in SearchResultSummaryDisplayContext

https://issues.liferay.com/browse/LPS-162241

Commit:
LPS-162241 portal-search-web: Add builds to ModifiedDate, ModifiedUserName, CreatedByUserPortraitURLString and ModifiedByUserPortraitURLString to searchSummaryDisplayContext

https://issues.liferay.com/browse/LPS-162241

jeffwu0724 committed:
LPS-142794 - add TransformerFactory newTransformerFactory() to SecureXMLFactoryProvider

jeffwu0724 committed:
LPS-142794 - add newTransformerFactory in SecureXMLFactoryProviderImpl and SecureXMLFactoryProviderUtil

Commit:
LPS-162241 portal-search: Add the statusByUserId in WorkflowedModelDocumentContributor to be indexed

https://issues.liferay.com/browse/LPS-162241

jeffwu0724 committed:
LPS-142794 - changed to use SecureXMLFactoryProviderUtil.newTransformerFactory() instead of TransformerFactory.newInstance() to avoid XXE vulnerability, please also see https://cheatsheetseries.owasp.org/cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.html#transformerfactory

Commit:
LPS-162241 portal-search-web: Fix buildModifiedUserPortrait and Edit buildModifiedUserName

https://issues.liferay.com/browse/LPS-162241

Commit:
LPS-162241 portal-search-web: Fix tests and Remove unnecessary code

https://issues.liferay.com/browse/LPS-162241

Commit:
LPS-162241 portal-search-web: Fix NullPointerException

https://issues.liferay.com/browse/LPS-162241

Commit:
LPS-162241 portal-search-web: Add templateVariables and Rename method in sumarryDisplayContextBuilder

https://issues.liferay.com/browse/LPS-162241

marianoalvarosaiz committed:
LPS-162111 Add test to ensure that groovy runtime exceptions thrown by the module can be serialized/deserialized.

jeffwu0724 committed:
LPS-142794 - use Mockito before test to set up _secureXMLFactoryProvider in SecureXMLFactoryProviderUtil and avoid NullPointerException

marianoalvarosaiz committed:
LPS-162111 Only the exception message and the stacktrace are needed, ignore other parameters like the class type.

Commit:
LPS-162241 Fix indexedFieldsTests

https://issues.liferay.com/browse/LPS-162241

Commit:
LPS-162241 portal-language-lang: Add Translation

https://issues.liferay.com/browse/LPS-162241

Commit:
LPS-162241 portal-language-lang: Regen

https://issues.liferay.com/browse/LPS-162241

Commit:
LPS-162241 portal-search-web: Fix string validation

https://issues.liferay.com/browse/LPS-162241

Commit:
LPS-162241 portal-search-web: Fix unit tests adding userId to the document

https://issues.liferay.com/browse/LPS-162429

julschong committed:
LPS-155670 Remove _getLibDir usage and move path normalization logic to where property is set

julschong committed:
LPS-155670 Remove _getLibDir method, no more usage

julschong committed:
LPS-155670 Remove _getLibDir usage and move path normalization logic to where property is set

Commit:
LPS-162473 Remove method and variable about ConfigurationAdmin in BaseConfigurationProvider

Commit:
LPS-162473 Get ConfigurationAdmin via method getConfigurationAdmin in CompanyScopedConfigurationProvider

Commit:
LPS-162473 Apply to subclasses

Commit:
LPS-162473 semantic versioning

arthurchan35 committed:
LPS-162500 Remove scheduler

BryanEngler committed:
LPS-162241 blogs-test: Remove duplicate put

https://issues.liferay.com/browse/LPS-162241

BryanEngler committed:
LPS-162241 portal-search-web: We already have creatorUser* so we dont need createdByUser*

https://issues.liferay.com/browse/LPS-162241

BryanEngler committed:
LPS-162241 portal-search-web: Use _getFieldValueLong

https://issues.liferay.com/browse/LPS-162241

BryanEngler committed:
LPS-162241 portal-search-elasticsearch7-impl: I dont think its necessary, but we should add a mapping

https://issues.liferay.com/browse/LPS-162241

arthurchan35 committed:
LPS-162500 Handle token refresh in non-blocking manner

arthurchan35 committed:
LPS-162500 Add an expiration date and finder

arthurchan35 committed:
LPS-162500 Auto gen

arthurchan35 committed:
LPS-162500 add upgrade process

arthurchan35 committed:
LPS-162500 make use of new field to avoid parsing

arthurchan35 committed:
LPS-162500 Instead of scheduling job per login, schedule a job for all

BryanEngler committed:
LPS-162241 portal-search-web: Put original logic back for setting UserPortrait

https://issues.liferay.com/browse/LPS-162241

julschong committed:
LPS-155670 Remove methods, variable, and unit test from ClassUtil, no more usage

Commit:
LPS-162726 Convert method references to field references

ces-hongvo committed:
LPS-161353 Implement URL validation accepRootRelative function

ces-hongvo committed:
LPS-161353 Using URL validation acceptRootRelative in DDMFormInstanceSettings

arthurchan35 committed:
LPS-162500 semver

BryanEngler committed:
LPS-162241 portal-search-web: Sort

https://issues.liferay.com/browse/LPS-162241

BryanEngler committed:
LPS-162241 portal-search-web: Rename labels

https://issues.liferay.com/browse/LPS-162241

BryanEngler committed:
LPS-162241 portal-language-lang: Rename lang keys and add help text

https://issues.liferay.com/browse/LPS-162241

BryanEngler committed:
LPS-162241 portal-language-lang: Sort

https://issues.liferay.com/browse/LPS-162241

BryanEngler committed:
LPS-162241 portal-search-web: SF

https://issues.liferay.com/browse/LPS-162241

BryanEngler committed:
LPS-162241 portal-search-web: userName is all lowercase, screenName can be alphanumeric whatever, so lets use userid + fullname

https://issues.liferay.com/browse/LPS-162241

BryanEngler committed:
LPS-162241 portal-search-web: Revert changes to the test since we are keeping the legacy portrait for backward compatibility

https://issues.liferay.com/browse/LPS-162241

marianoalvarosaiz committed:
LPS-162507 Move initialDeploymentUpgradeStep to ClientExtensionUpgradeStepRegistrator in order to be able to execute upgrade steps either on first install or when new upgrade steps are added.

ces-hongvo committed:
LPS-161353 Baseline packageinfo

susanchen3 committed:
LPS-162912 Add step to account for missing /

Commit:
LPS-162833 Remove method and variable about querySuggester, get it via method getQuerySuggester in BaseIndexSearcher

Commit:
LPS-162833 Apply to subclass, method setQuerySuggester is needed

Commit:
LPS-162833 Remove method and variable about spellCheckIndexWriter, get it via method getSpellCheckIndexWriter in BaseIndexWriter

Commit:
LPS-162833 Apply to subclass, method setSpellCheckIndexWriter is needed

Commit:
LPS-162833 Convert method references to field references

Commit:
LPS-162833 semantic versioning

ces-huynguyen committed:
LPS-162618 Deleting an Object when a Form is mapped to it breaks the Forms application

ces-huynguyen committed:
LPS-162618 Add languge key for form was created using deleted object

ces-huynguyen committed:
LPS-162618 buildLang

achaparro committed:
LPS-160370 Init default companyId as soon as possible during the startup

achaparro committed:
LPS-160370 Old logic is not longer needed. We must use the global method

achaparro committed:
LPS-160370 System and defaultCompany must have the same cache. We use 0 as key since we create the Ehcache before creating the first company during the initial startup, it is also useful to recognize the defaultCompany cache easily. We check if company is System first since PortalUtil is not ready until a company is added.

achaparro committed:
LPS-160370 We must pass the connection since the DataAccess is not ready at that point

LuisOrtiz89 committed:
LPS-95835 Added integration tests for UpgradeProcessFactory class

achaparro committed:
LPS-160370 When the defaultCompanyId is not initialized yet we must return 0 in case DB Partition

achaparro committed:
LPS-160370 Adding tests

julschong committed:
LPS-161324 Modify service builder persistence_impl.ftl and BasePersistenceImpl to use service reference

julschong committed:
LPS-161324 Auto, ant build-services

susanchen3 committed:
LPS-162002 Change version to use dxp

BryanEngler committed:
LPS-162241 portal-search-elasticsearch7-impl: Fix ElasticsearchIndexInformationTest.testGetFieldMappings test failure

https://issues.liferay.com/browse/LPS-162241

Commit:
LPS-159839 Add test JSONCanBeConfigured

LuisOrtiz89 committed:
LPS-95835 Adding default clause to column creation on test to avoid issues with DB2

achaparro committed:
LPS-160370 No longer needed

evibudai committed:
LPS-156654 Setting friendlyURLMapperClass on portletModel if it can be found in serviceReference

nNilton committed:
LPS-159613 update structuredContentFolder

nNilton committed:
LPS-159613 put externalReferenceCodes

Commit:
LPS-162241 portal-language-lang: buildLang

https://issues.liferay.com/browse/LPS-162241

Commit:
LPS-159839 Add macro configureJSON

Commit:
LPS-155518 Modify locator to adapt new behavior

Commit:
LPS-155518 Add locator for text input with aria label

Commit:
LPS-155518 Use new locator

Commit:
LPS-155518 Modify case to adapt new behavior

Commit:
LPS-155518 Modify macros to adapt new behavior

Commit:
LPS-155518 Simplify case

evibudai committed:
LPS-156368 Add new alert message language key

evibudai committed:
LPS-156368 Build Lang

evibudai committed:
LPS-156368 Show alert message when adding a documents and media field

Commit:
LPS-162241 portal-search-elasticsearch7-impl: Fix SearchResultsPortletTest.testDocumentWithoutSummaryIsRemoved test failure

https://issues.liferay.com/browse/LPS-162241

achaparro committed:
LPS-160370 Add a new method to avoid use PortalUtil in the cache module

achaparro committed:
LPS-160370 Baseline

achaparro committed:
LPS-160370 Avoid to catch an exception any time we create a new table on the first startup

Commit:
LPS-160627 Add AdminCanEditSuite Test

Commit:
LPS-160626 Add AdminCanDeleteSuite Test

Commit:
LPS-160574 Add AdminCanDesignSuite Test

Commit:
LPS-162815 Add AdminCanEditSmartSuite Test

Commit:
LPS-162815 Add AdminCanEditSmartSuite Test

Commit:
LPS-162816 Add AdminCanDeleteSmartSuite Test

Commit:
LPS-160627 Fix typeSuiteField macro to add description too

Commit:
LPS-160627 Add macros related to Suites

Commit:
LPS-160627 Add paths related to Suites

achaparro committed:
LPS-95835 SF Consistency

Commit:
LPS-156998 Fix AdminCanAddAndDeleteComponentWithoutTeam Test

Commit:
LPS-160627 SF

nNilton committed:
LPS-159613 assert by externalReferenceCode

nNilton committed:
LPS-159613 rename

Commit:
LPS-162241 portal-search-web: Add publishedDateString and publishedDateVisible and Generate getters and setters in SearchResultSummaryDisplayContex

https://issues.liferay.com/browse/LPS-162241

Commit:
LPS-162241 portal-search-web: Add method to build publishedDate field

https://issues.liferay.com/browse/LPS-162241

Commit:
LPS-162241 portal-search-web: Add templateVariable to publishedDate

https://issues.liferay.com/browse/LPS-162241

nNilton committed:
LPS-159613 rename article to folder

Commit:
LPS-162241 portal-language-lang: Add help text to published date

https://issues.liferay.com/browse/LPS-162241

Commit:
LPS-162241 portal-language-lang: BuildLang

https://issues.liferay.com/browse/LPS-162241

Commit:
LPS-162241 portal-search-web: Rename method to formatDate insted of formatCreationDate

https://issues.liferay.com/browse/LPS-162241

Commit:
LPS-155518 Modify macro to adapt new behavior

Commit:
LPS-155518 Modify cases to adapt new behavior

Commit:
LPS-155518 Modify macro to make it reusable

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

Jeremy-Ch committed:
LPS-163159 Add macro for the depot expot and import admin fast navigation

Jeremy-Ch committed:
LPS-163159 Split the tests into multiple tests

ling-alan-huang committed:
LPS-162237 Log exception instead of swallowing

ling-alan-huang committed:
LPS-162237 Log exception instead of swallowing

ling-alan-huang committed:
LPS-162237 Add exception to existing log call

ling-alan-huang committed:
LPS-162237 Log exception instead of swallowing

ling-alan-huang committed:
LPS-162237 Add exception to existing log call

ling-alan-huang committed:
LPS-162237 Add exception to existing log call

ling-alan-huang committed:
LPS-162237 Log exception instead of swallowing

ling-alan-huang committed:
LPS-162237 Log exception instead of swallowing

daiazius committed:
LPS-162788 Remove the Resend button when the notification is sent

tinatian committed:
LPS-162111 Rethrow ScriptingException instead of GroovyRuntimeException

jeffwu0724 committed:
LPS-142794 - auto semantic version

brianchandotcom committed:
artifact:ignore asset-browser-web 5.0.31 prep next

brianchandotcom committed:
artifact:ignore asset-categories-admin-web 5.0.57 prep next

brianchandotcom committed:
artifact:ignore asset-categories-navigation-web 6.0.22 prep next

brianchandotcom committed:
artifact:ignore asset-categories-selector-web 4.0.28 prep next

brianchandotcom committed:
artifact:ignore asset-display-web 5.0.13 prep next

brianchandotcom committed:
artifact:ignore asset-list-web 3.0.62 prep next

brianchandotcom committed:
artifact:ignore asset-publisher-web 5.0.76 prep next

brianchandotcom committed:
artifact:ignore asset-tags-admin-web 5.0.28 prep next

brianchandotcom committed:
artifact:ignore asset-tags-compiler-web 6.0.15 prep next

brianchandotcom committed:
artifact:ignore asset-tags-navigation-web 7.0.18 prep next

brianchandotcom committed:
artifact:ignore asset-tags-selector-web 4.0.21 prep next

brianchandotcom committed:
artifact:ignore asset-web 6.0.11 prep next

brianchandotcom committed:
artifact:ignore blogs-recent-bloggers-web 6.0.18 prep next

brianchandotcom committed:
artifact:ignore blogs-web 6.0.64 prep next

brianchandotcom committed:
artifact:ignore bookmarks-web 5.0.34 prep next

brianchandotcom committed:
artifact:ignore calendar-web 5.0.50 prep next

brianchandotcom committed:
artifact:ignore comment-page-comments-web 5.0.13 prep next

brianchandotcom committed:
artifact:ignore comment-web 6.0.17 prep next

brianchandotcom committed:
artifact:ignore document-library-video 1.0.44 prep next

brianchandotcom committed:
artifact:ignore document-library-web 6.0.106 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-lists-web 6.0.48 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-data-provider-web 5.0.26 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-web 5.0.59 prep next

brianchandotcom committed:
artifact:ignore item-selector-web 7.0.29 prep next

brianchandotcom committed:
artifact:ignore journal-web 5.0.94 prep next

brianchandotcom committed:
artifact:ignore layout-admin-web 5.0.99 prep next

brianchandotcom committed:
artifact:ignore message-boards-web 5.0.58 prep next

brianchandotcom committed:
artifact:ignore monitoring-web 5.0.16 prep next

brianchandotcom committed:
artifact:ignore trash-web 5.0.33 prep next

brianchandotcom committed:
artifact:ignore wiki-web 7.0.61 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-web 4.0.107 prep next

brianchandotcom committed:
artifact:ignore journal-content-web 6.0.36 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-forms-web 5.0.54 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-web 3.0.43 prep next

shuyangzhou committed:
LPS-163476 Since MFAEmailOTPConfiguration is default to disable, MFASetupUserAccountCheckerTracker should require configuration for activation.

Commit:
LPS-162833 Fix tests

victorg1991 committed:
LPS-159993 Adds a unitless version

The length field always save the value with an unit, but we may support older values that may no have the unit. This way if the value doesn't have the unit the first declaration will be invalid and the latter will take precedence

victorg1991 committed:
LPS-159993 Replace all values

victorg1991 committed:
LPS-159993 Auto SF

evibudai committed:
LPS-156368 SF

vicnate5 committed:
LPS-162912 Consistent variable name

Kvale1733 committed:
LRQA-75854 Ignore company logging tests until development is completed for LPS-154228

anabuchmann committed:
LRQA-74786 - Remove stub

ling-alan-huang committed:
LPS-147681 Add javax.portlet.version=3.0

shuyangzhou committed:
LPS-163489 Remove runtime validation, CI service builder batch is already ensuring the constants is up to date.

ling-alan-huang committed:
LPS-147681 Add javax.portlet.version=3.0

Luizcarlosqueiroz committed:
LRQA-77305 Fix test assertion

Luizcarlosqueiroz committed:
LRQA-77305 Remove unused path

shuyangzhou committed:
LPS-163489 Regenerate

john-co committed:
LPS-159839 Update with rebased macros

Commit:
LRQA-77312 Quarantined ProcessBuilderKaleoDesignerReact tests

john-co committed:
LPS-159839 SF

pyoo47 committed:
LRCI-3250 Check for tokens that are not found

pyoo47 committed:
LRCI-3250 Implement basic protection against invalid inputs

Luizcarlosqueiroz committed:
LRQA-77311 Improve title path

Luizcarlosqueiroz committed:
LRQA-77311 Fix test assertion

joelrocha68 committed:
LPS-161128 Fix CanViewEnvelopeDetails test

anabuchmann committed:
LRQA-74730 - Add CanConfigureActionAtTimers test

vicnate5 committed:
LPS-162912 Assertion change for new data archive

brianchandotcom committed:
LRCI-3250 SF

brianchandotcom committed:
LRCI-3250 prep next

anabuchmann committed:
LRQA-74730 - Add CanConfigureActionAtTimers test

brianchandotcom committed:
LPS-156368 Wordsmith

brianchandotcom committed:
LPS-156368 Regen

CAustin582 committed:
LPS-0 Use the CDN when downloading snapshots

Luizcarlosqueiroz committed:
LRQA-77311 Remove obsolete macro

brianchandotcom committed:
LPS-159649 prevent NPE

brianchandotcom committed:
artifact:ignore change-tracking-web 2.0.69 prep next

brianchandotcom committed:
LPS-161324 SF

brianchandotcom committed:
LPS-161324 prep next

brianchandotcom committed:
LPS-161324 prep next

brianchandotcom committed:
LPS-161324 regen

brianchandotcom committed:
Baseline

brianchandotcom committed:
LPS-156654 sort

Commit:
LPS-163322 Modify case to adapt new behavior

brianchandotcom committed:
artifact:ignore client-extension-service 1.0.23 prep next

brianchandotcom committed:
artifact:ignore depot-web 2.0.56 prep next

brianchandotcom committed:
artifact:ignore configuration-admin-web 5.0.46 prep next

brianchandotcom committed:
LPS-162111 SF

brianchandotcom committed:
LPS-162111 Rename

brianchandotcom committed:
LPS-162111 Rename

brianchandotcom committed:
LPS-162111 auto SF

Commit:
LPS-163487 Add step to close product menu

brianchandotcom committed:
LPS-163489 prep next

brianchandotcom committed:
LPS-163489 prep next

brianchandotcom committed:
LPS-162500 Wordsmith

brianchandotcom committed:
LPS-162500 Regen

brianchandotcom committed:
LPS-162500 Simplify/SF

brianchandotcom committed:
LPS-162500 SF

brianchandotcom committed:
LPS-162500 Add finder

brianchandotcom committed:
LPS-162500 Use generated finder

brianchandotcom committed:
LPS-162500 Regen

marianoalvarosaiz committed:
Revert "LPS-162507 Move initialDeploymentUpgradeStep to ClientExtensionUpgradeStepRegistrator in order to be able to execute upgrade steps either on first install or when new upgrade steps are added."

This reverts commit 3aec40058e14b9facc7cea672b7807f10f564477.

liferay-translation committed:
LPS-77699 Update Translations

achaparro committed:
LPS-160370 SF

achaparro commented:
@tinatian, I pushed a change to solve the conflicts.

achaparro commented:
Hi @tinatian 

Could you take a look at the pull? This is causing CI issues that avoid dbpartition suite to work and it is affecting LXC too.

Thank you.
